packages/server/lib/browsers/utils.coffee

```diff
@@ -1,9 +1,11 @@
 path     = require("path")
+debug    = require("debug")("cypress:server:browsers")
```

Comment:
somehow, some of these files have a 4th tag section and some don't... for the sake of eventual consistency, can this be used instead?

```suggestion
debug    = require("debug")("cypress:server:browsers:utils")
```